src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -234,111 +232,110 @@ public Node visitExprSingle(JsoniqParser.ExprSingleContext ctx) {
     // TODO [EXPRVISITOR] count
```

Comment:
Same with the TODO here

Reply:
I think that was also a leftover. The cloud clause is now supported.